osdteletext.c

```diff
@@ -340,6 +346,27 @@ cTeletextSetupPage::cTeletextSetupPage(void) {
    cString buf;
    cOsdItem *item;
 
+   //character set
+   static const char *txtBlock[] = {
```

Comment:
Sorry about the misleading comment of previous PR. I had a proper look at the code and your original way to include this into ```temp``` was correct and, please, change it as it were.